tests/h/nipsa/services_test.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import mock
+import pytest
+from pyramid.testing import DummyRequest
+
+from h.nipsa.models import NipsaUser
+from h.nipsa.services import NipsaService
+from h.nipsa.services import nipsa_factory
+
+
+@pytest.mark.usefixtures('nipsa_users', 'worker')
+class TestNipsaService(object):
+    def test_flagged_userids_returns_list_of_userids(self, db_session):
+        svc = NipsaService(db_session)
```

Comment:
Hmm. Maybe. Not sure about making the system-under-test a fixture unless constructing is particularly burdensome.